Parity/src/QwBPMStripline.cc

```diff
@@ -411,7 +413,7 @@ void  QwBPMStripline<T>::ProcessEvent()
       }
     }
   fEllipticity.Ratio(fEllipticity,fEffectiveCharge);
-  fEllipticity.Scale(1.0); // Include 2*k/sigma scale factor here
+  fEllipticity.Scale(0.5*18.81*18.81); // Include 2*k/sigma scale factor here
```

Comment:
Because I don't know what BSENfactor is I just bypassed it here.

Reply:
The BSENfactor for a particular BPM is specified in the geometry file, and is defined to be "true calibration"/18.81.  The calibration is coming from the effective scale size of the BPM can itself, and so we probably should be using the calibration with the BSENfactor applied.

Reply:
The "Scale" parameter should be "0.5*fQwStriplineCalibrationfQwStriplineCalibration"